source/default-conf.py

```diff
@@ -250,7 +250,17 @@
 # Add https://www.min.io/robots.txt to html_extra_path list once available.
 html_extra_path = [ 'extra']
 
-html_title = 'MinIO Object Storage for ' + ("MacOS" if platform == "macos" else platform.capitalize())
+platform_fmt = ""
+
+if platform == "macos":
+   platform_fmt = "MacOS"
+elif platform == "openshift":
+   platform_fmt = "OpenShift"
+else:
+   platform_fmt = platform.capitalize()
```

Comment:
Needed so that things are customized in the left navbar under "MinIO Object Storage for Blah"